Jenkinsfile

```diff
@@ -232,25 +232,42 @@ timestamps {
     }
   }
 
-  // Local nodes use os: windows | macos; Remote tests use os full name e.g.: macOS Sonoma
-  def platforms = [
-    // Local tests
-    // [ browser: 'edge', os: 'windows' ],
-    // [ browser: 'firefox', os: 'macos' ],
-    // Remote tests
-    // [ browser: 'chrome', provider: 'aws', buckets: 2 ],
-    // [ browser: 'edge', provider: 'aws', buckets: 2 ], // TINY-10540: Investigate Edge issues in AWS
-    // [ browser: 'firefox', provider: 'aws', buckets: 2 ],
-    [ browser: 'chrome', provider: 'lambdatest', buckets: 1 ],
-    [ browser: 'firefox', provider: 'lambdatest', buckets: 1 ],
-    [ browser: 'edge', provider: 'lambdatest', buckets: 1 ],
-    [ browser: 'chrome', provider: 'lambdatest', os: 'macOS Sequoia', buckets: 1 ],
-    [ browser: 'firefox', provider: 'lambdatest', os: 'macOS Sequoia', buckets: 1 ],
-    [ browser: 'safari', provider: 'lambdatest', os: 'macOS Sequoia', buckets: 1 ]
+  // Local nodes use os: windows | macos;
+  // Remote tests use os full name e.g.: macOS Sonoma
+
+  // Local tests (never do this, it depends on jenkins nodes)
```

Comment:
Should we remove this entire chunk. All of it is remote anyway now days right.

Reply:
Yeah. Even "local" testing should be done with selenium containers, which we have in the premium repo, not with Jenkins nodes.